src/components/routes/Member/AuthenticatedMember/AuthenticatedMember.jsx

```diff
@@ -17,9 +17,11 @@ class AuthenticatedMember extends React.Component {
     const { config } = this.props;
     if (!isEmpty(config)) {
       return (
-        <div className={"authenticated-mbr-container"}>
+        <div className={'authenticated-mbr__container'}>
```

Comment:
As per the eslint the correct way to provide className is 
```suggestion
        <div className="authenticated-mbr__container">
```

Reply:
done